core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/SenderThreadsBackOffManager.java

```diff
@@ -83,8 +83,8 @@ public synchronized void stopAllSendersBackOffActivities() {
 
     @Override
     protected SenderThreadLocalBackOffData initialValue() {
-        int addSeconds = threadsSecondsDifference.incrementAndGet();
-        senderThreadLocalData = new SenderThreadLocalBackOffData(backOffTimeoutsInMilliseconds, addSeconds * 1000);
+        long addSeconds = threadsSecondsDifference.incrementAndGet();
```

Comment:
this cast to long is not needed, is it for clarity?
```suggestion
        int addSeconds = threadsSecondsDifference.incrementAndGet();
```

Reply:
The constructor is as shown:
https://github.com/microsoft/ApplicationInsights-Java/blob/a4437aa0817797e9e19e0f4ed7156290c9b22644/core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/SenderThreadLocalBackOffData.java#L55

The multiplication of `addSeconds * 1000` was flagged because it's `int * int` could overflow if `addSeconds` is sufficiently large (I don't think it's possible in this case, but...). Changing the type seemed like the most straightforward way to address it.

Reply:
I suppose `addSeconds` could be inlined and the literal changed to `long`. That's probably cleaner

Reply:
I see what you meant now. I forgot the literal was already changed to `long` 😄